tests/wildcard.rs

```diff
@@ -77,7 +49,8 @@ fn wildcard() {
 
 #[test]
 fn invalid_segment_error() {
-    let mut server = make_server();
+    let app = tide::App::new(());
```

Comment:
We need to set up routers for *all* test functions, as functions can't see each other.

Reply:
I added the routes. I wonder if there is a better way of initializing `app` and add the routes, so every test `fn` can access it...